tests/e2e/backup_restore_suite_test.go

```diff
@@ -104,6 +105,14 @@ var _ = Describe("AWS backup restore tests", func() {
 				Skip(reason)
 			}
 
+			if provider == "azure" && brCase.BackupRestoreType == CSI {
+				availability, _ := version.NewVersion("4.10")
```

Comment:
can we make this >=("4.10") before we move on?

Reply:
For anything >= 4.10 will not  hit the skip() statement. I do not think we need >=4.10 condition.